lib/tasks/evm_dba.rake

```diff
@@ -100,37 +100,13 @@ namespace :evm do
     end
 
     desc "Seed the ManageIQ EVM Database (VMDB) with defaults"
-    task :seed do
-      Rake::Task['db:seed'].invoke
-    end
+    task :seed => "db:seed"
```

Comment:
👍

Reply:
Not quite sure why we have this.  Maybe the caller should instead call db:seed directly?  I'm not sure why this is here.  Maybe we had additional logic at some point?

Reply:
We had this for consistency - we were making the use do `evm:` prefix for everything else, so for consitency, this one also had an evm: prefix.

Reply:
pulled out into #22003